Python/1. print.md

```diff
@@ -44,6 +44,20 @@ print("Hoi", "BOI")
 Hoi BOI
 ```
 
+### Code
+
+```python
+# Adjust the sep value to change what goes between strings
+# " " (space) is default
```

Comment:
hmmm maybe make that not a comment and text instead. BTW ima change all these things if u r not online rn